### PR TITLE
Siri extension

### LINKED ISSUE #3047: Create the Siri extension scaffold
Apparently there's some problem creating the intents without the extension. So this issue needs to create the extension and add the Siri capability to the project.

We don't need to add the handling of the intents just yet, having the empty extension should suffice for now.

### PR DESCRIPTION
**Pull request by Odrakir**
## What's this?
Creates the extension to handle Siri Intents. It doesn't actually have the code to handle them yet. It also adds Siri capability to the app.

### Relationships
This is required to create the intents in #3042, #3046 and the rest.

Closes #3047 

## How is it done?
This is done through Visual Studio and it creates a new project (inside Daneel)

### Side effects
Development provisioning profile has been changed.

## :squid: Permissions
As long as you squash into `siri-shorcuts` we are ok.

### CODE REVIEW

File: Toggl.sln [hunk 1]
```diff
@@ -495,6 +497,46 @@ Global
...
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release.AdHoc|Any CPU.ActiveCfg = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release.AdHoc|Any CPU.Build.0 = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release.PlayStore|Any CPU.ActiveCfg = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release.PlayStore|Any CPU.Build.0 = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release.AdHoc.Giskard|Any CPU.ActiveCfg = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release.AdHoc.Giskard|Any CPU.Build.0 = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.UITests|Any CPU.ActiveCfg = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.UITests|Any CPU.Build.0 = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.SyncDiagramGenerator|Any CPU.ActiveCfg = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.SyncDiagramGenerator|Any CPU.Build.0 = Debug|iPhoneSimulator
```
Comment (amulware) [inline Toggl.sln]: These are way too many build configs. Please make sure this is only built when actually needed.

File: SiriExtension/SiriExtension.csproj [hunk 2]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">iPhoneSimulator</Platform>
+    <ProjectGuid>{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}</ProjectGuid>
+    <ProjectTypeGuids>{EE2C853D-36AF-4FDB-B1AD-8E90477E2198};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <OutputType>Library</OutputType>
+    <RootNamespace>SiriExtension</RootNamespace>
+    <AssemblyName>SiriExtension</AssemblyName>
```
Comment (khoi) [inline SiriExtension/SiriExtension.csproj]: Probably name the project `Toggl.Daneel.SiriExtension` for consistency

**Comment by amulware** [comment]
There is still a bunch of configurations that make no sense to me, from tests to sync graph builder. :/

File: Toggl.sln [hunk 3]
```diff
@@ -495,6 +497,28 @@ Global
 		{148C791B-9824-4DC6-9494-4FEEB21988C7}.UnitTests|Any CPU.Build.0 = Debug|Any CPU
 		{148C791B-9824-4DC6-9494-4FEEB21988C7}.SyncDiagramGenerator|Any CPU.ActiveCfg = Debug|Any CPU
 		{148C791B-9824-4DC6-9494-4FEEB21988C7}.SyncDiagramGenerator|Any CPU.Build.0 = Debug|Any CPU
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.ApiTests|Any CPU.ActiveCfg = Debug|iPhoneSimulator
```
Comment (amulware) [inline Toggl.sln]: api tests?

File: Toggl.sln [hunk 4]
```diff
@@ -495,6 +497,28 @@ Global
...
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Debug|iPhoneSimulator.ActiveCfg = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Debug|iPhoneSimulator.Build.0 = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release.AppStore|Any CPU.ActiveCfg = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release.AppStore|Any CPU.Build.0 = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release|Any CPU.ActiveCfg = Release|iPhone
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release|Any CPU.Build.0 = Release|iPhone
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release|iPhone.ActiveCfg = Release|iPhone
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release|iPhone.Build.0 = Release|iPhone
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release|iPhoneSimulator.ActiveCfg = Release|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release|iPhoneSimulator.Build.0 = Release|iPhoneSimulator
```
Comment (amulware) [inline Toggl.sln]: didn't know we had a Release config (six lines)? Is that being used for anything?

File: Toggl.sln [hunk 5]
```diff
@@ -495,6 +497,28 @@ Global
...
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.UnitTests|Any CPU.ActiveCfg = Debug|iPhoneSimulator
```
Comment (amulware) [inline Toggl.sln]: unit tests?

File: Toggl.sln [hunk 6]
```diff
@@ -495,6 +497,28 @@ Global
...
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release.AdHoc|Any CPU.ActiveCfg = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release.AdHoc|Any CPU.Build.0 = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release.PlayStore|Any CPU.ActiveCfg = Debug|iPhoneSimulator
+		{48C970FE-E13F-4664-A2FD-4EBFD2DD131E}.Release.AdHoc.Giskard|Any CPU.ActiveCfg = Debug|iPhoneSimulator
```
Comment (amulware) [inline Toggl.sln]: playstore and giskard adhoc builds?

File: Toggl.sln (hunk 4 above)
Comment (Odrakir) [reply Toggl.sln]: No idea. I'll delete it and see if I break something.

**Pull request merged**